### ISSUE TITLE
test results

### ISSUE THREAD

**Issue by Zizzzzzzz** [eventbasedvision/eTraM#9]
Hello! When I tested on the test set using the RVT weights you provided, I obtained the following results, which do not match the results reported in your paper. What could be the reason?
class1:
![image](https://github.com/eventbasedvision/eTraM/assets/88081108/71fd2a4e-3f29-4897-85df-afee2b977e5d)
class2:
![image](https://github.com/eventbasedvision/eTraM/assets/88081108/e7e3a637-f340-461b-a7b5-cdd5428cbc04)
class3:
![image](https://github.com/eventbasedvision/eTraM/assets/88081108/2352c25d-e38f-4aff-99aa-c62ec2384d9d)
all:
![image](https://github.com/eventbasedvision/eTraM/assets/88081108/06b1cfa8-4451-431d-9248-7e37a7bb1669)

**Comment by aayush-v** [comment]
Hello @Zizzzzzzz thank you for reaching out. The pre-trained weights we provide are mainly for reference purposes. Additionally, please note that the results reported in the paper are at IoU=50.
